HLTriggerOffline/Btag/python/HltBtagValidation_cff.py

```diff
@@ -19,17 +19,17 @@
         SimVertexCollection = cms.InputTag("g4SimHits"),
 	TriggerResults = cms.InputTag('TriggerResults','',"HLT"),
 	HLTPathNames =cms.vstring(
-	'HLT_PFMET120_',
-	'HLT_PFMET120_',
-	'HLT_PFMET120_',
-	'HLT_HT300PT30_QuadJet_75_60_45_40_v',
-	'HLT_HT300PT30_QuadJet_75_60_45_40_v',
-	'HLT_HT300PT30_QuadJet_75_60_45_40_v',
-	'HLT_PFHT380_SixJet32_DoubleBTagCSV_',
-	'HLT_PFHT380_SixJet32_DoubleBTagCSV_',
-	'HLT_PFHT380_SixJet32_DoubleBTagCSV_',
+	'HLT_PFMET120_PFMHT120_IDTight_v',
+	'HLT_PFMET120_PFMHT120_IDTight_v',
+	'HLT_PFMET120_PFMHT120_IDTight_v',
+	'HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v',
+	'HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v',
+	'HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v',
+	'HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_',
```

Comment:
shouldn't it be
`HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_v`

Reply:
was this question resolved? @natalia-korneeva

Reply:
@davidlange6 yes as @mtosi agrees with my point and doesn't insist on finishing with _v